frontend/device/pocketbook/device.lua

```diff
@@ -37,8 +37,10 @@ local PocketBook = Generic:new{
     -- all devices that have warmth lights use inkview api
     hasNaturalLightApi = yes,
 
-    -- Apparently, HW inversion is a pipedream on PB (#6669)
-    canHWInvert = no,
+    -- NOTE: Apparently, HW inversion is a pipedream on PB (#6669), ... well, on sunxi chipsets anyway.
+    -- For which we now probe in fbinfoOverride() and tweak the flag to "no".
+    -- NXT/other chipsets *should* work (PB631), but in case it doesn't on your device, set this to "no" in here.
```

Comment:
Nitpick: It's NTX, not NXT ;).

(And the SoCs are from NXP (ex-Freescale), which doesn't help avoid confusion ^^).